apiMethods/commentsMethods.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
skup sie na artykulach, zrobmy to dobrze, i potem bedziesz robil kolejne moduly, bo inaczej bedzie syf.